airflow/www/templates/airflow/gantt.html

```diff
@@ -79,6 +79,6 @@
   <script src="{{ url_for_asset('gantt.js') }}"></script>
   <script>
     // Loading data via <meta> wasn't loading extra_links in a task
-    const data = JSON.parse('{{ data|tojson }}');
+    const data = JSON.parse('{{ data|tojson|replace("\\", "\\\\") }}');
```

Comment:
While I see the problem, this always sound suspicious if you want to make such individual replaces. 

Is \\ the only possible problem here? Aren't there ready to use filters we can use to sanitize the input "fully"?

Reply:
I think \ is only problem here because every other character has already be escaped by \ , now, we need to escape the \ that is used to escape other character. There is [forceescape()](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.forceescape) filter available but it will escape doublequote more suitable for html rather than javacript string.

Reply:
Manual replace does not look right to me. We could likely fix this by putting the string in HTML instead and use `getElementById` or something to retrieve the value here.